bindings/matrix-sdk-ffi/src/sliding_sync.rs

```diff
@@ -289,6 +324,18 @@ impl SlidingSyncViewBuilder {
         Arc::new(builder)
     }
 
+    pub fn filters(self: Arc<Self>, filters: SlidingSyncRequestListFilters) -> Arc<Self> {
+        let mut builder = unwrap_or_clone_arc(self);
+        builder.inner = builder.inner.filters(Some(filters.into()));
+        Arc::new(builder)
+    }
+
+    pub fn no_filters(self: Arc<Self>) -> Arc<Self> {
+        let mut builder = unwrap_or_clone_arc(self);
+        builder.inner = builder.inner.filters(None);
+        Arc::new(builder)
+    }
```

Comment:
You don't seem to be declaring these in UDL. Also I don't see any features used here that the proc-macro couldn't handle, so you can likely just derive `uniffi::Record` for `SlidingSyncRequestListFilters` and put these two methods in an impl block with `#[uniffi::export]`.